agile-development/team-agreements/team-manifesto/readme.md

```diff
@@ -0,0 +1,78 @@
+
+The difference of team manifesto from other comprehensive set of standards/practices documents is, it is used to give a short summary of the upcoming way of working and practices to prepare a base before deep dive starts with open discussion environment in an ice-breaker way.
+
+Below, you can find some including, but not limited, topics many teams touch during engagements,
+
+| Topic | What is it about ? |
+|-|-|
+| Collective Ownership | Does team own the code rather than individuals? And what is the expectation? |
+| Respect | Any preferred statement about it's a "must-have" team value |
+| Collaboration | Any preferred statement about how does team want to collaborate ? |
+| Transparency | A simple statement about it's a "must-have" team value and if preferred, how does this being provided by the team ? meetings, retrospective, feedback mechanisms ..etc. |
+| Craftpersonship | Which tools such as Git, VS Code LiveShare, ..etc. are being used ? What is the definition of expected best usage of them? |
+| PR sizing | What does team prefer in PRs ? |
+| Branching | Team's branching strategy and standards |
+| Commit standards | Preferred format in commit messages, rules and more |
+| Clean Code | Does team follow clean code principles ? |
+| Pair/Mob Programming | Will team apply pair/mob programming ? If yes, what programming styles are suitable for the team ? |
+| Release Process | Principles around release proicess such as quality gates, reviewing process ...etc. |
+| Code Review | Any rule for code reviewing such as min number of reviewers, team rules ...etc. |
+| Action Readiness | How the backlog will be groomed ? What will be the way for clear DoD and ACs ? |
+| TDD | Will team follow TDD ? |
+| Test Coverage | Is there any expected number, percentage or measurement ?  |
+| Dimensions in Testing | Required tests for high quality software, eg : unit, integration, functional, performance, regression, acceptance |
+| Build process | build for all? or not; The clear statement of where code and under what conditions code should work ? eg : OS, DevOps, tool dependency |
+| Bug fix | The rules of bug fixing in the team ? eg: contact people, attaching PR to the issue ..etc. |
+| Technical debt | How does team manage/follow it?  |
+| Refactoring | How does team manage/follow it? |
+| Agile Documentation | Does team want to use diagrams and tables more rather than detailed KB articles ? |
+| Efficient Documentation | When is it necessary ? Is it a prerequisite to complete tasks/PRs ..etc.? |
+| Definition of Fun | How will we have fun for relaxing/enjoying the team spirit during the engagement? |
+
+## Tools
+
+Generally team sessions are enough for building a manifesto and having a consensus around it.
+And if there is a need for improving it in a structured way, [This guy, That guy*](https://www.scrum.nl/blog/building-team-manifesto/) or other retrospective tools can be used.
```

Comment:
Probably just go with "Building a Team Manifesto" rather than "This guy, That guy"

Reply:
fixed commit b07b8657c0ea24e73211d9bfc93994abe9cf3545